### PR TITLE
[IMP][account_refund_early_payment] create one refund and the account…

### PR DESCRIPTION
**Pull request by JulioSerna**
… receivable is split by the same invoices to reconcile each invoice with only refund.

And migration to new Api.

### CODE REVIEW

File: account_refund_early_payment/wizard/refund_early_payments.py [hunk 1]
```diff
@@ -88,43 +75,44 @@ def _get_percent_default(self):
...
+    @api.multi
+    def compute_refund(self, mode):
+        context = dict(self._context)
 
-        result = super(AccountInvoiceRefund, self).compute_refund(
-            cur, uid, ids, mode, context=context)
+        if mode == 'early_payment':
+            ctx = context.copy()
+            context.update(
+                {'active_ids': context.get('active_ids')[0]})
```
Comment (moylop260) [inline account_refund_early_payment/wizard/refund_early_payments.py:89]: Is possbile a `active_ids` empty or not defined?

File: account_refund_early_payment/view/refund_early_payment.xml [hunk 2]
```diff
@@ -16,5 +16,15 @@
...
         </record>
+
+        <act_window name="Create refund invoice"
+                res_model="account.invoice.refund"
+                src_model="account.invoice"
+                view_mode="form"
+                target="new"
+                key2="client_action_multi"
+                id="act_wizard_refund_invoice"
+                context="{'ctx_active_ids': active_ids, 'ctx_active_model':active_model}"
```
Comment (nhomar) [inline account_refund_early_payment/view/refund_early_payment.xml]: **tontería** coloca bien los espacios en el dict, despues del segundo `:` hace falta.

File: account_refund_early_payment/wizard/refund_early_payments.py [hunk 3]
```diff
@@ -139,50 +127,79 @@ def compute_refund(self, cur, uid, ids, mode='refund', context=None):
...
+            amount_total_inv += inv.amount_total
+            invoice_source.append(inv.number)
+
+        for inv in inv_obj.browse(context.get('active_ids')):
+            for line in inv.move_id.line_id:
+                if line.account_id.reconcile and not\
+                        line.reconcile_id and\
+                        line.account_id == move_line_id_refund.account_id:
+                    amount_inv_refund = (
+                        inv.amount_total / amount_total_inv) *\
```
Comment (moylop260) [inline account_refund_early_payment/wizard/refund_early_payments.py]: Is posible a divizion error?

**Comment by hbto** [comment]
+1

**Pull request merged**